src/compiler/checker.ts

```diff
@@ -16188,6 +16196,11 @@ namespace ts {
             return result && symbolName(result);
         }
 
+        function getSuggestionForNonexistentModule(name: Identifier, targetModule: Symbol): string | undefined {

```

Comment:
one small nit: `getSuggestionForNonexistentExport` might be a better name. Not sure if it's worth changing through.